aten/src/ATen/native/cuda/AccumulateType.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
```

Comment:
In https://github.com/pytorch/pytorch/pull/4304, I started putting utility-style cuda code in /cuda rather than /native/cuda.  I think this makes sense because technically everything that isn't a TH/THC wrapper is "native", so we need to have a more precise definition of native to limit what goes in /native; I'd argue functional extensions are a natural definition.  Since this seems generally useful and isn't just implementing a functional extension, I'd put it in /cuda.

Reply:
Yeah, I like that better. I'll rebase and move it there after you merge your PR.